bundle/manifests/openshift-builds-sample-buildah-external-registry.yaml

```
@@ -0,0 +1,34 @@
+apiVersion: console.openshift.io/v1
+kind: ConsoleYAMLSample
+metadata:
+  name: buildah-build-external-push
+spec:
+  description: A sample Build using buildah BuildStrategy that pushes the built image to an external registry
+  snippet: false
+  targetResource:
+    apiVersion: shipwright.io/v1beta1
+    kind: Build
+  title: Buildah Build External Push
+  yaml: |
+    apiVersion: shipwright.io/v1beta1
+    kind: Build
+    metadata:
+      name: buildah-golang-build-external-push
+    spec:
+      source:
+        type: Git
+        git:
+          url: https://github.com/redhat-openshift-builds/samples/
+        contextDir: buildah-build
+      strategy:
+        name: buildah
+        kind: ClusterBuildStrategy
+      paramValues:
+      - name: dockerfile
+        value: Dockerfile
+      output:
+        # For a successful image push, a pushSecret with relevant credentials must be available in the same namespace as builds.
+        # Detailed steps are available at: https://docs.openshift.com/builds/1.1/authenticating/understanding-authentication-at-runtime.html#ob-authentication-to-container-registries_understanding-authentication-at-runtime
```

This should point to `docs.redhat.com` - `docs.openshift.com` is being decomissioned soon.
```suggestion
        # Detailed steps are available at: https://docs.redhat.com/en/documentation/builds_for_red_hat_openshift/1.1/html/authentication/understanding-authentication-at-runtime#ob-authentication-to-container-registries_understanding-authentication-at-runtime
```